doc/develop/languages/c/minimal_libc.rst

```diff
@@ -33,6 +33,90 @@ The minimal libc uses the malloc api family implementation provided by the
 :ref:`common C library <c_library_common>`, which itself is built upon the
 :ref:`kernel memory heap API <heap_v2>`.
 
+.. _c_library_minimal_functions:
+
+Functions
+*********
+
+The minimal libc implements only the minimal subset of the standard C library
```

Comment:
I wonder if it would be reasonable to allow any ISO/IEC C library function to be used by Zephyr code so that developers weren't encouraged to re-implement them slightly incorrectly on their own? A patch adding use of a new function would need to include an implementation for use in the minimal C library. This seems aligned with the goal of continuing to support external ISO/IEC compliant toolchains while not hindering use of well defined standard functions.

Reply:
> I wonder if it would be reasonable to allow any ISO/IEC C library function to be used by Zephyr code so that developers weren't encouraged to re-implement them slightly incorrectly on their own?

That is the goal of Rule A.5, which says "Zephyr codebase shall not use the functions and macros that are not defined by the ISO C standard" (i.e. it does not say Zephyr codebase only needs to use the ISO C standard functions that are implemented by the Zephyr minimal libc -- that only applies to the "Zephyr kernel," which is a small subset of Zephyr).

I will try to reword the Rule A.5 to make this more clear.

Reply:
Done (hopefully).